src/mem.rs

```diff
@@ -0,0 +1,10 @@
+#![no_main]
+#![no_std]
+
+extern crate pwasm_std;
+
+#[no_mangle]
+pub fn call(desc: *mut u8) {
```

Comment:
maybe add something that ensures that memory operations are actually used
for example, modify static region with length 10k+ with input arguments or smth?